manual_tests/scenarios/draft_pr.py

```diff
@@ -23,16 +19,19 @@ def test(
         sh.banner("Wait for the terraform_plan check to complete")
         assert pr.check("Terraform Plan").wait(pr.since).success
 
-        # The terraform_lock check should not run automatically when the PR is a draft
-        sh.banner("Make sure the terraform_lock check did not run")
-        assert pr.check("Terraform Lock").wait(pr.since).skipped
+        # The slices should not be locked
+        sh.banner("Make sure the slices are not locked")
+        for slice in slices:
+            assert not slice.is_locked()
 
         # Since this PR is a draft, it should not be able to apply the plan
         sh.banner("Try to apply the plan for a draft PR")
         since = pr.add_label(":taco::apply")
 
         # The terraform_apply check should not run automatically when the PR is a draft
-        assert pr.check("Terraform Apply").wait(pr.since).skipped
+        assert (
+            pr.check("Terraform Apply", "tacos_apply").wait(pr.since).skipped
+        )
```

Comment:
This assertion doesn't quite do what it's meant to.
The intent is to show that tacos apply doesn't run after the label is added.
Instead this asserts that was skipped after the pr was created, _at any time_.
The fix is simple:

```suggestion
        assert (
            pr.check("Terraform Apply", "tacos_apply").wait(since).skipped
        )
```